data/net/minecraft/world/level/levelgen/structure/BoundingBox.mapping

```diff
@@ -21,35 +23,44 @@ CLASS net/minecraft/world/level/levelgen/structure/BoundingBox
 	METHOD forAllCorners (Ljava/util/function/Consumer;)V
 		ARG 1 pos
 	METHOD fromCorners (Lnet/minecraft/core/Vec3i;Lnet/minecraft/core/Vec3i;)Lnet/minecraft/world/level/levelgen/structure/BoundingBox;
-		ARG 0 minVec
-		ARG 1 maxVec
+		ARG 0 first
+		ARG 1 second
+	METHOD getCenter ()Lnet/minecraft/core/BlockPos;
+		COMMENT @return The center of this bounding box. Note in even-sized dimensions the center position will be offset in the positive direction.
+	METHOD getLength ()Lnet/minecraft/core/Vec3i;
+		COMMENT Returns a vector describing the dimensions of this bounding box.
+		COMMENT Note that unlike {@code getXSpan()}, {@code getYSpan()}, and {@code getZSpan()}, the length is interpreted here as the difference in coordinates. So a box over a 1x1x1 area, which still contains a single point, will report length zero.
 	METHOD getXSpan ()I
-		COMMENT Get dimension of the bounding box in the x direction.
+		COMMENT @return The length of this bounding box along the x axis.
 	METHOD getYSpan ()I
-		COMMENT Get dimension of the bounding box in the y direction.
+		COMMENT @return The length of this bounding box along the y axis.
 	METHOD getZSpan ()I
-		COMMENT Get dimension of the bounding box in the z direction.
+		COMMENT @return The length of this bounding box along the z axis.
 	METHOD inflate (I)Lnet/minecraft/world/level/levelgen/structure/BoundingBox;
+		COMMENT Expands this box by a fixed {@code value} in all directions.
 		ARG 1 value
 	METHOD intersects (IIII)Z
-		COMMENT Discover if a coordinate is inside the bounding box area.
+		COMMENT @return true if this bounding box inersects the horizontal x/z region described by the min and max parameters.
 		ARG 1 minX
 		ARG 2 minZ
 		ARG 3 maxX
 		ARG 4 maxZ
 	METHOD intersects (Lnet/minecraft/world/level/levelgen/structure/BoundingBox;)Z
-		COMMENT Discover if bounding box can fit within the current bounding box object.
-		ARG 1 structurebb
+		COMMENT @return true if {@code box} intersects this box.
+		ARG 1 box
 	METHOD isInside (Lnet/minecraft/core/Vec3i;)Z
-		COMMENT Checks if given Vec3i is inside of StructureBoundingBox
-		ARG 1 vec
+		COMMENT @return true if the bounding box contains the {@code vector}
+		ARG 1 vector
 	METHOD move (III)Lnet/minecraft/world/level/levelgen/structure/BoundingBox;
+		COMMENT Translates this box by the given coordinates.
```

Comment:
Possibly consider explicitly declaring the return value of this value being the same instance (to note that this modifies the values in the instance), as opposed to `#moved` which returns a new instance without modifying the original.